default_app/main.js

```diff
@@ -93,25 +90,25 @@ app.once('ready', function () {
         {
           label: 'Reload',
           accelerator: 'CmdOrCtrl+R',
-          click: function (item, focusedWindow) {
+          click(item, focusedWindow) {
             if (focusedWindow) focusedWindow.reload()
           }
         },
         {
           label: 'Toggle Full Screen',
-          accelerator: (function () {
+          accelerator: (() => {
             return (process.platform === 'darwin') ? 'Ctrl+Command+F' : 'F11'
           })(),
-          click: function (item, focusedWindow) {
+          click(item, focusedWindow) {
             if (focusedWindow) focusedWindow.setFullScreen(!focusedWindow.isFullScreen())
           }
         },
         {
           label: 'Toggle Developer Tools',
-          accelerator: (function () {
+          accelerator: (() => {
             return (process.platform === 'darwin') ? 'Alt+Command+I' : 'Ctrl+Shift+I'
           })(),
-          click: function (item, focusedWindow) {
+          click(item, focusedWindow) {
```

Comment:
Thanks—I really like it, too. That said, the linting process in CI doesn't like it—even though it's valid code. Do you think I should change the code to appease the linter?